src/components/NetworkSelector/index.test.tsx

```diff
@@ -81,12 +92,12 @@ describe('NetworkSelector', () => {
     })
 
     test('it changes the network', async () => {
-      const { getByText, queryByText } = render(
-        <TestEnvironment>
+      const { getByText } = render(
+        <EnvironmentProvider>
           <ThemeProvider theme={theme}>
             <NetworkSelector />
           </ThemeProvider>
-        </TestEnvironment>
+        </EnvironmentProvider>
```

Comment:
What's the reason for wrapping with `EnvironmentProvider` and not `TestEnvironment`?

Reply:
The `TestEnvironment` doesn't (on its own) export `setNetwork` since the EnvironmentProvider does that now

Reply:
Ah right. So `Environment` just provides the variables and for stories and tests we don't need more than that I assume. Is that the main reason those two are separated? Would there be something against just ditching `Environment` and only using `EnvironmentProvider`? Just curious.

Reply:
Yeah thats how i've been thinking about it - 

In terms of TestEnvironment using the `EnvironmentProvider` - I think this would be ideal (and something we should work towards), but it's a bigger operation since the current tests mock `useRouter` in different ways, so would need to think about how to consolidate or standardize that mocking. 

Trying to avoid scope creep on this PR so we can get the warn-duplicate stuff in